vllm_gaudi/extension/bucketing/linear.py

```diff
@@ -64,6 +64,11 @@ def get_decode_cfgs(self, max_num_seqs, block_size, max_num_batched_tokens, max_
                 f'VLLM_DECODE_BLOCK_BUCKET_MAX={decode_block_bucket_cfg[2]} is higher than max_blocks={max_blocks}. Your configuration VLLM_DECODE_BLOCK_BUCKET_MAX={decode_block_bucket_cfg[2]} will be overwritten to VLLM_DECODE_BLOCK_BUCKET_MAX={max_blocks}'
             )
             decode_block_bucket_cfg[2] = max_blocks
+            if decode_block_bucket_cfg[0] > max_blocks:
+                logger().info(
+                    f'VLLM_DECODE_BLOCK_BUCKET_MIN={decode_block_bucket_cfg[0]} is higher than max_blocks={max_blocks}. Your configuration VLLM_DECODE_BLOCK_BUCKET_MIN={decode_block_bucket_cfg[0]} will be overwritten to VLLM_DECODE_BLOCK_BUCKET_MIN={max_blocks - decode_block_bucket_cfg[1]}'
+                )
+                decode_block_bucket_cfg[0] = max_blocks - decode_block_bucket_cfg[1]
```

Comment:
do we need also to check that this value is >0 ?

Reply:
Added a clip